File: WEEKLYCODE#1/friends_and_birthday_parties/Java/friends_and_birthday_parties.java

import java.util.*;
import java.lang.*;
import java.io.*;
import java.util.Scanner;

public class solution
{
    public static void main (String[] args) throws java.lang.Exception
    {
        Scanner input = new Scanner(System.in);
        int res=1,i;
        String str;
        str = input.nextLine();
        //calculating lenght of the string
        int len=str.length();
        //converting string to character array
        char s[] = str.toCharArray(); 
        //counting number of commas (,)
         for(i=0;i<len;i++)   
                if(s[i]==',')   
                    res++;  
        
        System.out.println(res);
    }
}

File: WEEKLYCODE#1/friends_and_birthday_parties/C/friends_and_birthday_parties.c
#include < stdio.h >
#include < string.h >
#include < string.h >
#define MAX_STRING_LENGHT 1001
int main(){

    char str[MAX_STRING_LENGHT];
    int i,lenght,count=0;
    //we take string as input
    scanf("%s",str);
    //calculating the length of the string
    lenght=strlen(str);
    //counting the number of commas(,) in the string
    for(i=0;i<lenght;i++){
        if(str[i]==','){
            count++;
        }
    }
    //printing answer
    printf("%d\n",count+1);
    return 0;
}
```
